apidoc/Titanium/Media/Alert.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Titanium.Media.Alert
+summary: An object for playing system sounds.
+description: |
+    You can use this module to provide audible alerts. 
+
+    You can use it to play short (30 seconds or shorter) sounds. The interface does not provide level, positioning, looping, or timing control, and does not support simultaneous playback: You can play only one sound at a time. 
+
+    This module differs from the Sound module because it honors the ringtone volume, not the Music volume.
+
+    Use the <Titanium.Media.createAlert> method to create a `Alert` object.
+
+    Know more about [System Sound Services](https://developer.apple.com/reference/audiotoolbox/1657326-system_sound_services).
+
```

Comment:
Remove the empty lines here, it's formatted as a paragraph by the docs-generator.